src/ucs/datastruct/arbiter.h

```diff
@@ -113,7 +113,9 @@ typedef enum {
 #define UCS_ARBITER_GROUP_ARBITER_SET(_group, _arbiter) \
     (_group)->arbiter = (_arbiter)
 #define UCS_ARBITER_GROUP_ARBITER_CHECK(_group, _arbiter) \
-    ucs_assert((_group)->arbiter == (_arbiter))
+    ucs_assertv((_group)->arbiter == (_arbiter), \
+                "%p == %p", (_group)->arbiter, _group)
+#define UCS_ARBITER_GROUP_ARBITER(_group) (_group)->arbiter
```

Comment:
why do we need this macro UCS_ARBITER_GROUP_ARBITER() ?

Reply:
to take arbiter from dummy group and assign it to the currently dispatched group

Reply:
i'm pretty sure we can pass it directly w/o this macro